Pareto/AppHandlers.swift

```diff
@@ -335,10 +335,14 @@ class AppHandlers: NSObject, NetworkHandlerObserver {
     func copyLogs() {
         if !SystemUser.current.isAdmin {
             let alert = NSAlert()
-            alert.messageText = "Logs cannot be copied to the clipboard. Instead, you must use the Console app to capture the application logs by searching for “ParetoSecurity” and copying them to the clipboard."
+            alert.messageText = "Logs cannot be copied to the clipboard because you are using a non-admin macOS account. Instead, you must use the Console app to capture the application logs by searching for \"Pareto Security\" and copying them to the clipboard."
```

Comment:
```suggestion
            alert.messageText = "Logs cannot be copied to the clipboard because you are using a non-admin user account. Please use the Console app to capture the application logs by searching for \"Pareto Security\" and copying them to the clipboard."
```

A bit shorter